libsupport/src/JSONTracer.cpp

```diff
@@ -114,15 +114,15 @@ std::string
 BuildJSON(
     const std::string& trace_id, const std::string& span_data,
     const std::string& log_data, const std::string& tag_data,
-    const std::string& host_data) {
+    const std::string& host_data, uint64_t duration) {
   fmt::memory_buffer buf;
   uint32_t host_id = katana::ProgressTracer::Get().GetHostID();
   static auto begin = katana::Now();
   auto msec_since_begin = katana::UsSince(begin) / 1000;
 
   fmt::format_to(
-      std::back_inserter(buf), R"({{"host":{},"offset_ms":{})", host_id,
-      msec_since_begin);
+      std::back_inserter(buf), R"({{"host":{},"offset_ms":{},"span_ms":{})",
```

Comment:
I get how this makes it easier to read the raw traces, but why wouldn't we just make the tools that parse traces easier to use?

Reply:
@l-hoang is the primary asker for this feature

Reply:
And the offset_ms fields are also just cruft to make the logs easier to use and aren't used to assemble traces